libathemecore/email.c

```diff
@@ -317,6 +317,17 @@ sendemail(struct user *u, struct myuser *mu, const char *type, const char *email
 			u->nick, u->user, u->vhost, u->ip ? u->ip : u->host,
 			type, entity(mu)->name, email);
 
+	struct hook_sendemail_req hdata = {
```

Comment:
I know that `.handled` will default initialize to `false` here, but I think it'd be nice to be explicit about that as we're initializing all the other fields.